test-configs.yaml

```
@@ -679,6 +679,11 @@ device_types:
     dtb: 'freescale/imx8mp-evk.dtb'
     boot_method: uboot
 
+  imx8mq-zii-ultra-zest:
+    mach: imx
+    arch: arm64
```

You need to define the `dtb` name, especially as it's in a directory different from the `mach` name:

```suggestion
    arch: arm64
    dtb: freescale/imx8mq-zii-ultra-zest.dtb
```

---

updated!

---

Great, let's see tomorrow if staging jobs worked as expected.